### PR TITLE
Podcasts: Add Exposed Links for Social Sharing

### PR DESCRIPTION
**Pull request by jestapinski**
[Staging Link](https://docs-mongodborg-staging.corp.mongodb.com/master/devhub/jordanstapinski/add-exposed-links/)

This PR mimics #804 to expose the links on the sides for the podcast pages. I also made sure to keep the tooltip text customizable. This way, if this project is ready for production first it can be deployed without the need for the devhub feedback project to be ready.

### CODE REVIEW

File: src/components/dev-hub/controlled-tooltip.js [hunk 1]
```diff
@@ -4,7 +4,10 @@ import { css } from '@emotion/react';
 import Popover from 'react-tiny-popover';
 import { animationSpeed, layer, size } from './theme';
 
-const ControlledTooltipWrapper = styled('span')`
```
Comment (Dmytro-Shustra-EPAM) [inline src/components/dev-hub/controlled-tooltip.js:7]: Screenshot 2021-05-17 at 23 43 28
Some components are using `span`  for CSS nesting and add margins let's move this logic here.

File: src/templates/podcast.js [hunk 2]
```diff
@@ -83,6 +80,29 @@ const StyledPlayer = styled(AudioPlayer)`
...
+    align-items: center;
+    > :not(:first-child) {
+        margin-top: ${size.medium};
+    }
+    @media ${screenSize.upToLarge} {
+        display: inline-flex;
+        flex-direction: row;
+        > * {
+            margin-top: 0;
+            margin-left: ${size.mediumLarge};
```
Comment (Dmytro-Shustra-EPAM) [inline src/templates/podcast.js:94]: Screenshot 2021-05-17 at 23 53 54
Looks like we don't need `margin-left` for the first element, let's omit first element.

File: src/templates/podcast.js [hunk 3]
```diff
@@ -83,6 +80,29 @@ const StyledPlayer = styled(AudioPlayer)`
     }
 `;
 
+const StyledBlogShareLinks = styled(BlogShareLinks)`
+    flex-direction: column;
+    align-items: center;
+    > :not(:first-child) {
+        margin-top: ${size.medium};
```
Comment (Dmytro-Shustra-EPAM) [inline src/templates/podcast.js:87]: Screenshot 2021-05-17 at 23 58 28
Also, here we need this margins only for screen 1024px and hire.

File: src/templates/podcast.js [hunk 4]
```diff
@@ -83,6 +80,29 @@ const StyledPlayer = styled(AudioPlayer)`
...
+        > * {
+            margin-top: 0;
+            margin-left: ${size.mediumLarge};
+        }
+    }
+`;
+
+const StyledFooter = styled(ShareFooter)`
+    /* Target the copy link, although it is below some divs */
+    div > div > a {
```
Comment (Dmytro-Shustra-EPAM) [inline src/templates/podcast.js:101]: Like nesting in CSS it's good, but we can import the styled components and use nesting with them, I prefer this way, but you can omit this suggestion.

**Review by Dmytro-Shustra-EPAM — COMMENTED** [review]
Good so far! Some CSS is not exact for sharing links at the top (article, podcast). I have left comments, let's check this out icons position and responsive.

**Review by Dmytro-Shustra-EPAM — APPROVED** [review]
Awesome work! Thanks, @jestapinski!

**Comment by jestapinski** [comment]
Thanks for the feedback @Dmytro-Shustra-EPAM !

**Pull request merged**